### DISCUSSION TITLE
How to help with es-cl translation

### DISCUSSION THREAD

**Discussion by Vynzaro** [vectorwing/FarmersDelight#1028] [General]
### Minecraft version

1.20.1

### Farmer's Delight version

1.2.6

### Description

My mother language is _chilean_ spanish, and also speak spanish. 
I'd like to help in _chilean_ spanish translation.
(I know this description was so vague. Sorry :c)

**Comment by vectorwing** [comment]
Hey there! I moved this issue to Discussions, as it's not related to problems with the mod.

We usually receive new translations through a Pull Request (PR for short). An example:
https://github.com/vectorwing/FarmersDelight/pull/1026

However, if neither you or your mother are familiar with git or GitHub, you can simply send me an `es_cl.json` directly, and I can analyze it myself. This can be done through here, or through a venue such as Discord.

Let me know if you have other questions! 👍

**Reply by Vynzaro** [reply]
Idk where send the .json so... here is:
https://pastebin.com/kSpNLeyR

This proposal has idioms and things from Chilean culture in the achievements and another stuff. I decided to propose the idioms in the achievements as a way of Chilean culture in the translation and I based myself on the example of one of the minecraft achievements "they are pololos" when you mate two animals. I find it cute to put idioms from each dialect.
An example of the idioms I put is "¡Pone la mesa!" or put the cutlery and plates, becouse in Chile when someone that is cooking (usually your dad/mom/grandma) tells you that the food is served, sometimes it isn't true and the _Chilean mom/dad_ tells you that you have to set the table before, that is, it is not served yet.

**Reply by vectorwing** [reply]
Sorry for the late reply! We ended up receiving a PR for the es_cl translation 20 days after this, and I forgot to apply your changes at the time. My bad! 😓

I can try and diff your translations against the new one we have. But if you'd like, you can also review the current one and see if it's accurate.
